src/js/veteran-id-card/components/RequiredVeteranView.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SystemDownView from '../../common/components/SystemDownView';
+
+class RequiredVeteranView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true
+    };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ loading: false });
+    }, 2000);
```

Comment:
Since this uses the same state and is wrapped in RequiredLoginView which displays the loading indicator, I'm just tearing out any loading-related logic.